src/harness/harness.ts

```diff
@@ -502,7 +502,7 @@ namespace Harness {
         tryEnableSourceMapsForHost?(): void;
         getEnvironmentVariable?(name: string): string;
     }
-    export let IO: IO;
+    export let IO: Io;

```

Comment:
This is C# naming, not JavaScript naming.

Reply:
This isn't a public interface, thus won't affect external users.
You might want to make a pull request to tslint if this still bothers you. Then you could make a pull request here once that's published.

Reply:
Naming conventions are important for collaboration, maintainability.

I don't think your workaround would work. My suggestion would be to place `// tslint:disable-next-line` on it.

Reply:
Why don't you think making a pull request to tslint would work?

Reply:
Late to the party, but we aren't using `IO` as an ***I*nterface prefix here for an `O` type, but `IO` meaning **I**nput/**O**utput, so this rule should have been disabled here rather than enforced on the `IO` and `IOLog` interfaces. It's also cumbersome that we have `Harness.IO` the value and `Harness.Io` the type.